DemoServer/Controllers/Demos/StaticIndexes/StoreFieldsOnIndex/metadata.json

```diff
@@ -0,0 +1,248 @@
+                },
+                {
+                    "Title": "Static Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/10-static-indexes-and-other-advanced-options#static-indexes-and-other-advanced-options"
+                },
+                {
+                    "Title": "Working with Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/12-working-with-indexes#working-with-indexes"
+                }
+            ]
+        },
+        {
+            "Title": "The index Entry class",
+            "Slug": "step-2",
+            "DescriptionLines": [
+                "* The IndexEntry class defines the _index-fields per entry_ in the index.<br>These fields will be indexed and can be queried on.",
+                "",
+                "* The _content_ for each index-field is configured in the index constructor (see next step).",
+                "",
+                "* Field _TotalItemsOrdered_ will be computed and stored on the index (Step 5)."
+            ],
+            "Assets": [
+                {
+                    "Title": "Map Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/csharp/indexes/map-indexes"
+                },
+                {
+                    "Title": "Static Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/10-static-indexes-and-other-advanced-options#static-indexes-and-other-advanced-options"
+                },
+                {
+                    "Title": "Working with Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/12-working-with-indexes#working-with-indexes"
+                }
+            ]
+        },
+        {
+            "Title": "The projected class",
+            "Slug": "step-3",
+            "DescriptionLines": [
+                "* `OrderProjectedDetails` will contain the details projected per document result.<br>",
+                "  Used in step 6.",
+                "",
+                "* Note: it is not mandatory for this class to be nested within the index class."
+            ],
+            "Assets": [
+                {
+                    "Title": "Project Index Result Demo",
+                    "Type" : "demo",
+                    "Url"  : "/static-indexes/project-index-results"
+                }
+            ]
+        },
+        {
+            "Title": "The Index Map",
+            "Slug": "step-4",
+            "DescriptionLines": [
+                "* The _Map function_ defines the index-fields content - which fields from the 'Order' entity will compose each index-field.",
```

Comment:
'Order' entity - would `'Order' document` be a slightly more precise formulation, since this is happening on the RavenDB server-side, where documents have not been converted to entities yet?

Reply:
right - done